sympy/simplify/tests/test_simplify.py

```diff
@@ -1036,6 +1036,22 @@ def test_issue_23543():
     x, y, z = symbols("x y z", commutative=False)
     assert (x*(y + z/2)).simplify() == x*(2*y + z)/2
 
+def test_issue_11004():
```

Comment:
Two blank lines needed between functions.

Reply:
This referred to functions not closures. There should be two blank lines before the `def test_issue_11004():` line and a single one afterwards (before the `    def f(n):` line).